test/tls-provider.c

```diff
@@ -41,41 +41,134 @@ typedef struct xorkey_st {
     int haspubkey;
 } XORKEY;
 
-/* We define a dummy TLS group called "xorgroup" for test purposes */
 
-static unsigned int group_id = 0; /* IANA reserved for private use */
```

Comment:
See 316e1e5558ca4a559a869262b55743661956ad92

Reply:
> I inherited this comment from your code ;)

Yeah. I think originally the actual group number was in the code - using a group number from the private use range. Later in the review process it was changed to randomly assign a value from the private range - so the comment no longer made sense.

I think we can just drop the comment from the group_id declaration because it is explained already in the randomisation code.

Reply:
Would yo prefer to remove the reference to `provider-base(7)` as the current version?

I can do it, not an issue, but it has the added value of reminding readers (or the writer on this side of the screen, as I always struggle to find them) where to look for the definition of those fields (and why those types).

Reply:
No - its fine. Leave it there.